v22.1/api-support-policy.md

```diff
@@ -0,0 +1,89 @@
+| [`cockroach` commands](cockroach-commands.html)                         | Mixed    | Versioned concurrently with CockroachDB.                                               |                                                                                                                                                              | All products                                                      |
+| [Health Endpoints](monitoring-and-alerting.html#health-endpoints)       | Stable   | No new versions forthcoming.                                                           |                                                                                                                                                              | All products                                                      |
+| [Prometheus Endpoint](monitoring-and-alerting.html#prometheus-endpoint) | Stable   | No new versions forthcoming.                                                           | Although this endpoint is not versioned, individual metrics may be added or removed in each CockroachDB release. No changes are expected to response format. | {{ site.data.products.dedicated }}, {{ site.data.products.core }} |
+| [Cluster API](cluster-api.html)                                         | Stable   | [Versioned independently from CockroachDB.](cluster-api.html#versioning-and-stability) |                                                                                                                                                              | {{ site.data.products.dedicated }}, {{ site.data.products.core }} |
+| [DB Console](ui-overview.html)                                          | Unstable | N/A                                                                                    | For stable access to the information present in this tool, use the [Cluster API](cluster-api.html).                                                          | {{ site.data.products.dedicated }}, {{ site.data.products.core }} |
+| [Logging](logging-overview.html)                                        | Mixed    | Versioned concurrently with CockroachDB.                                               | Stability varies by [event type](eventlog.html). Structured events are stable and unstructured events are unstable.                                          | {{ site.data.products.core }}                                     |
+| `ccloud` CLI                                                            | Mixed    | Versioned independently from CockroachDB.                                              | Default output is unstable. Specify the `–json` argument in the CLI for stable output that follows the versioning scheme.                                    | {{ site.data.products.db }}                                       |
```

Comment:
Not sure if you would want to link to the Get Started docs for `ccloud`? https://www.cockroachlabs.com/docs/cockroachcloud/ccloud-get-started.html. Might not work here